ui/app/clinical/consultation/views/treatmentSections/drugOrderHistory.html

```diff
@@ -66,14 +81,14 @@
                                         </span>
                                         <span class="provider">
                                             <provider-directive creator-name="{{drugOrder.creatorName}}" provider-name="{{drugOrder.provider.name}}"></provider-directive>
-                                        </span>
+                                         </span>
                                     </div>
                                 </div>
                                 <div ng-if="drugOrder.isMarkedForDiscontinue">
                                     <div class="form fl drug-stop-container">
                                         <article class="form-field">
                                             <div class="field-attribute stop-date-label">
-                                                <label class="control-label">{{ ::'DRUG_DETAILS_STOP_DATE' | translate }}</label>
+                                                <label class="control-label">{{ ::'DRUG_DETAILS_DATE' | translate }}</label>
```

Comment:
This change was made for testing, Just revert as it is to `DRUG_DETAILS_STOP_DATE`